<launch>
	<!-- <node pkg="cv_camera" type="cv_camera_node" name="camera" /> -->
	<node pkg="urg_node" type="urg_node" name="daart_hokuyo" />
	<node pkg="daart" type="daart_odom_node" name="daart_odom_node" />
	<node pkg="daart" type="daart_speed_control_node" name="daart_speed_control_node" />
	<node pkg="razor_imu_9dof" type="imu_node.py" name="imu_node" output="screen">
		<rosparam file="$(find daart)/config/razor.yaml" command="load"/>
	</node>

</launch>
